src/delta_lake/delta_service.py

```diff
@@ -115,6 +118,74 @@ def _check_exists(database: str, table: str) -> bool:
     return True
 
 
+def _generate_cache_key(prefix: str, params: Dict[str, Any]) -> str:
```

Comment:
I don't think I understand how this is going to work - I assumed we'd want to have a cache from SQL query -> result, in which case the cache key would need to be based on the SQL string

Reply:
not necessarily a query-to-result relationship, but rather endpoint parameters to result.

Reply:
:+1:

Reply:
Although I can see us possibly wanting to generalize this in the future